src/components/Person/Person.jsx

```diff
@@ -1 +1,26 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import './Person.scss';
+
+const personStatus = (sex) => {
+  const partner = sex === 'm' ? 'wife' : 'husband';
+
+  return partner;
+};
+
+export const Person = ({ person }) => (
```

Comment:
Please descruct your props, it will be more cleaner
```suggestion
export const Person = ({ person }) => (
  const {
    name,
    sex,
    ...
  } = person
```